creator-node/src/services/sync/syncUtil.ts

```diff
@@ -138,5 +177,33 @@ async function fetchExportFromNode({
   }
 }
 
-export { fetchExportFromNode }
-export type { FetchExportParams, FetchExportOutput }
+async function getSyncStatus(syncUuid: string) {
+  return redisClient.get(`syncStatus${syncUuid}`) as Promise<SyncStatus | null>
+}
+
+async function setSyncStatus(
+  syncUuid: string,
+  syncStatus: SyncStatus,
+  syncType: string
+) {
+  await redisClient.set(
+    `syncStatus${syncUuid}`,
+    syncStatus,
+    'PX',
+    getMaxSyncMonitoringMs(syncType) + SYNC_MONITORING_RETRY_DELAY_MS + 1
```

Comment:
k 2 requests could we 1. define this value outside of here with a comment as to the reasoning and 2. throw a v short comment for 'PX' bc i gotta look it up and i'm sure so will everyone else haha

Reply:
ah another thing - the sending and receiving node could in theory have different monitoring intervals which would further break this logic

Reply:
all solid points - I added that comment and changed it to just set expiry to an hour since no sync should ever need to monitor for anywhere near an hour (it's 45sec manual and 5min recurring as of now)

Reply:
dope